patches/components-search_engines-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/search_engines/BUILD.gn b/components/search_engines/BUILD.gn
+index edf777981961409f4e198042546078b0fa1dc922..e886ebaab539ddb200b4dbfd84a14108c3da329d 100644
+--- a/components/search_engines/BUILD.gn
++++ b/components/search_engines/BUILD.gn
+@@ -126,6 +126,7 @@ static_library("search_engines") {
+       "//components/search_engines/android:features_jni_headers",
+       "//components/search_engines/android:jni_headers",
+     ]
++    deps += ["//brave/components/search_engines/android:jni_headers"]
```

Comment:
What is the reason it is here? Shouldn't it be in our component?

Reply:
we can remove the patch altogether. Previously the changes were included in `search_engines` package. so we had to add deps for it.

Reply:
it;s resolved here : https://github.com/brave/brave-core/pull/27768/commits/df0fd26d97046ab7418d1604676e4d57580625ca